package.json

```diff
@@ -6,22 +6,32 @@
   "description": "State management for React Native",
   "license": "MIT",
   "private": true,
+  "files": [
+    "dist"
+  ],
+  "exports": {
+    "./web": "dist/index.web.js",
+    ".": "dist/index.native.js"
+  },
   "scripts": {
     "lint": "eslint .",
     "lint-tests": "eslint tests/**",
     "test": "jest",
-    "build-docs": "node buildDocs.js"
+    "build": "npm run build:native && npm run build:web",
+    "build:docs": "node buildDocs.js",
+    "build:native": "webpack --config webpack.native.js",
+    "build:web": "webpack --config webpack.web.js"
   },
   "dependencies": {
     "ascii-table": "0.0.9",
-    "expensify-common": "git+https://github.com/Expensify/expensify-common.git#2e5cff552cf132da90a3fb9756e6b4fb6ae7b40c",
```

Comment:
Specifying `expensify-common` under peer dependencies ensures both `Expnesify/App` and `Expnsify/react-native-onyx` use the same `expensify-common` version (the one from App)